From dddd76800c46372693a3e8e8e9f8142614d677c6 Mon Sep 17 00:00:00 2001
From: Alberto Milone <alberto.milone@canonical.com>
Date: Thu, 24 Oct 2013 13:28:47 +0200
Subject: [PATCH 1/1] Do not complain that the NVIDIA driver is not in use

Suggesting nvidia-xconfig won't help, as we autoload the
driver anyway. That would also break hybrid graphics.
---
 src/gtk+-2.x/ctkui.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gtk+-2.x/ctkui.c b/src/gtk+-2.x/ctkui.c
index 3ee395f..69b67c7 100644
--- a/src/gtk+-2.x/ctkui.c
+++ b/src/gtk+-2.x/ctkui.c
@@ -56,7 +56,7 @@ void ctk_main(ParsedAttribute *p,
             break;
         }
     }
-
+#if 0
     if (!has_nv_control) {
         GtkWidget *dlg;
         dlg = gtk_message_dialog_new (NULL,
@@ -70,7 +70,7 @@ void ctk_main(ParsedAttribute *p,
         gtk_dialog_run(GTK_DIALOG(dlg));
         gtk_widget_destroy (dlg);
     }
-
+#endif
     ctk_window_set_active_page(CTK_WINDOW(window), page);
 
     gtk_main();
-- 
1.7.9.5

